_docs/Instructions-Compile.md

```diff
@@ -1,94 +1,97 @@
 - [Azure Storage Explorer](https://azure.microsoft.com/en-us/features/storage-explorer/) (Easy way to upload and download files (see Local Emulator Database)
+- [Azure Data Studio](https://azure.microsoft.com/en-us/products/data-studio/) (Not required, can use IDE Tools for DB Querying)
 
 - Install Dev Tunnels or Ngrok see the rule https://ssw.com.au/rules/port-forwarding/
-- [dev tunnels](https://learn.microsoft.com/en-us/azure/developer/dev-tunnels/get-started?tabs=macos) (Recommended)
-- [ngrok](https://ngrok.com/) (Api and Mobile communication)
-****
+  - [dev tunnels](https://learn.microsoft.com/en-us/azure/developer/dev-tunnels/get-started?tabs=macos) (Recommended)
+  - [ngrok](https://ngrok.com/)
 
 ### Required Tools (for Mac)
 
 - XCode v15.0.1+
   - Check or set `Command Line Tools` location
   - **Preferences** | **Locations** | **Command Line Tools**
-- Visual Studio for Mac 2022 (EOL - April 2024)
-  - Enable .NET 8 SDK in **Preferences** | **Other** | **Preview Features**
-- [Azurite](https://learn.microsoft.com/en-us/azure/storage/common/storage-use-azurite?tabs=visual-studio,blob-storage) (For Docker or Visual Studio Code)
-
-## The F5 Experience
-
-### Web API
-
-1. Clone the repository 
-   1. `git clone https://github.com/SSWConsulting/SSW.Rewards.Mobile.git`
-3. Grab connection strings from an existing dev to create local `secrets.json` file (https://docs.microsoft.com/en-us/aspnet/core/security/app-secrets?view=aspnetcore-3.0&tabs=windows)
-4. Update `WebAPI` `appsettings.json` and `appsettings.development.json` with the proper secrets
-5. Install / Run `Azure Storage Emulator` (Run the .exe or use the "start" command)
-6. Install / Run [Azure Storage Explorer](https://azure.microsoft.com/en-us/features/storage-explorer/#overview)
-7. Start Explorer (**On First Install** - See steps below)
-   1. Sign in using the "Subscription" option
-   2. Select the "Azure" option
-   3. Next (Sign in with your account that has access to ssw1 DevOps)
-8. Inside the Explorer - **Local & Attached** | **Storage Accounts** | **Emulator** | **Blob Containers**
-9. Create a **new container** called `profile`
-10. Copy over all files in `SeedData/profiles` (images + .xlsx) into the newly created `profile` container
-11. Press f5
-12. Test that you can access the swagger docs @ `https://localhost:5001/swagger/`
-
-**(Optional)** - If you need to run the api and mobile app locally. 
-1. Run dev tunnels `devtunnel host -p 5001`
+
+## Setting up the Repo for Development
+### To work on the API + Admin UI
+1. Pull this Repo
+2. Grab the Secrets from Keeper 
+   1. **Client Secrets | SSW | SSW.Rewards | Developer Secrets**
+   2. Add them as .NET User Secrets for `WebAPI.csproj`
+   3. Update `appsettings.*.json` files accordingly
+3. Create a Developer Certificate https://learn.microsoft.com/en-us/aspnet/core/security/docker-https?view=aspnetcore-8.0#certificates
```

Comment:
Can this not be done as part of an `up.ps1` script?

Reply:
That would replace the next couple of steps

Reply:
Take a look at the coding challenge repo as an example 

https://github.com/SSWConsulting/SSW.CodingChallengeReimagined/blob/main/up.ps1

Reply:
@GordonBeeming,

Great suggestion, I've created a new issue https://github.com/SSWConsulting/SSW.Rewards.Mobile/issues/588